requirements.txt

```diff
@@ -11,4 +11,4 @@ moto==1.3.7
 Pillow>=3.3.1
 numpy>=1.11.1
 intern>=0.9.7
-
+enum34
```

Comment:
What's the earliest version of python we're planning on supporting? I don't know if we need this. [3.4 is already EOL](https://www.python.org/downloads/release/python-3410/) and [it had enum support](https://docs.python.org/3.4/library/enum.html). (Even 3.5 EOLs in september of this year)

Reply:
Didn't know 3.4 was EOL! Guess we don't need this requirement in `intern` either, then.